adjust/src/main/java/com/rudderstack/android/integration/adjust/AdjustIntegrationFactory.java

```diff
@@ -76,13 +75,15 @@ private AdjustIntegrationFactory(Object config, final RudderClient client, Rudde
                 apiToken,
                 rudderConfig.getLogLevel() >= RudderLogger.RudderLogLevel.DEBUG ? AdjustConfig.ENVIRONMENT_SANDBOX : AdjustConfig.ENVIRONMENT_PRODUCTION
         );
-        setLogLevel(rudderConfig, adjustConfig);
-
+        Utils.setLogLevel(rudderConfig, adjustConfig);
         adjustConfig.setOnAttributionChangedListener(new OnAttributionChangedListener() {
             @Override
             public void onAttributionChanged(AdjustAttribution attribution) {
-                Log.d("AdjustFactory", "Attribution callback called!");
-                Log.d("AdjustFactory", "Attribution: " + attribution.toString());
+                RudderLogger.logInfo("AdjustFactory: Attribution callback triggered - sending Install Attributed event");
+                RudderLogger.logDebug("AdjustFactory: Attribution details: " + attribution.toString());
+
+                // Send Install Attributed event to RudderStack
+                Utils.sendInstallAttributedEvent(client, attribution);
```

Comment:
Let’s add a TODO comment for the attribution flag, which allows users to enable or disable tracking.